include/sound/sof/info.h

```diff
@@ -109,9 +109,9 @@ struct sof_ipc_cc_version {
 	/* reserved for future use */
```

Comment:
@xiulipan Minor: "align with Linux style" This is not really true. This is not a Linux style issue, but of correct operation. C standard only says "char" is at least 8bits, it says nothing of its maximum size. So if you have a binary interface and you need to describe a variable of 8bits, you must use uint8_t or similar.

Reply:
@kv2019i got that, so I should said to make a ABI, I should use uint8_t instead of char. Thanks for explanation

Reply:
@xiulipan Sorry to nitpick but I'd still change the commit message 
"replace char to have fixed length for string"
-> into:
"replace char in packed ABI structs"

... the string lengths are already fixed without the patch, but size of the struct can vary.